packages/web-components/src/components/va-checkbox-group/test/va-checkbox-group.e2e.ts

```diff
@@ -22,7 +22,7 @@ describe('va-checkbox-group', () => {
     `,
     );
 
-    await axeCheck(page);
+    await axeCheck(page, ['aria-allowed-role']);
```

Comment:
>  I had to add some "ignore rule" statements to the AxeCheck tests.

Do we think there might be any issues in vets-website tests where `cy.axeCheck();` is being called?

Reply:
I don't know, but it's possible. Would the vets-website axe checks pierce the shadow dom?

Reply:
@jamigibbs I ran many of the vets-website Cypress tests with this update and didn't receive any axe check errors.